R/rows.R

```diff
@@ -8,7 +8,17 @@
 #' When `in_place = TRUE` these verbs do not generate `SELECT` queries, but
 #' instead directly modify the underlying data using `INSERT`, `UPDATE`, or
 #' `DELETE` operators. This will require that you have write access to
-#' the database.
+#' the database: the connection needs permission to insert, modify or delete
+#' rows, but not to alter the structure of the table.
+#'
+#' The default, `in_place = FALSE`, generates equivalent lazy tables (using
+#' `SELECT` queries) that allow previewing the result without actually
+#' modifying the underlying table on the database.
+#'
+#' @details
+#' For `in_place = TRUE`, the `x` argument must refer to a table instantiated
+#' with [tbl()], not to a lazy query. The [remote_name()] function is used
```

Comment:
I think this is fine though not 100% correct:
* one can do something like `tbl(con, "SELECT ... FROM ...")` which does not work (I just noticed `remote_name()` gives a name for such a table but it is quite an edge case anyway...)
* it also works after `compute()`

But I think it is clear enough as it is.